playbook-website/app/javascript/site_styles/_opacity_guidelines.scss

```diff
@@ -0,0 +1,48 @@
+ul {
+   
+  &.opacity-play {
+  list-style: none;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin: none;
+  background: none;
+  border-radius: none;
+  border: 0px;
+  padding: none;
+  flex-wrap: wrap;
```

Comment:
Indendation
```suggestion
    list-style: none;
    display: flex;
    justify-content: flex-start;
    align-items: center;
    margin: none;
    background: none;
    border-radius: none;
    border: 0px;
    padding: none;
    flex-wrap: wrap;
```

Reply:
@cvargaspower did you install the VSCode extension `Beautify`?

<img width="410" alt="Screen Shot 2021-11-19 at 8 46 57 AM" src="https://user-images.githubusercontent.com/2293844/142641702-5cb72b2d-8351-4ee6-b56f-a3ab022d2e3b.png">

If not, you will need to do that now. On your working files, you should right click and choose "Format Document" and these things (as mentioned above) will auto-fix.